eu-north-1/dev/ec2.tf

```diff
@@ -0,0 +1,121 @@
+#import vpc module
+#import vpc module
+module "vpc_module" {
+  source = "../../modules/vpc"
+  vpc_cidr = "10.0.0.0/16"
+  vpc_name = "custom vpc"
+  vpc_stack_owner = "data platform team"
+  vpc_stack_environment = "dev" 
+  subnet_cidr = "10.0.0.0/24"
+  subnet_name = "public subnet"
+}
+
+
+module "vpc_module_priv" {
+  source = "../../modules/vpc"
+  vpc_cidr = "10.0.0.0/16"
+  vpc_name = "custom vpc"
```

Comment:
You are creating 2 VPC with tyhe same name ?

Reply:
why do you need 2 VPC

Reply:
I have already modified this. I will make a pull request as soon as possible